proto/control.proto

```diff
@@ -11,12 +11,15 @@ service PDPControl {
 message Item {
   enum DataType {
     POLICIES = 0;
-    CONTENT  = 1;
+    CONTENT = 1;
+    POLICIES_PATCH = 2;
+    CONTENT_PATCH = 3;
   }
-           DataType type     = 1;
-           int32    dataId   = 2;
-           string   id       = 3;
-  repeated int32    includes = 4;
+  DataType type = 1;
+  string version = 2;
```

Comment:
I see here only one version field and now think that it is version of PDP's data we try update. However it's not enough we also need to tell PDP to which version this update switches PDP.